index.rst

```diff
@@ -100,6 +95,8 @@ Additionally, EditorConfig defines the following terms:
 - Section Name: the string between the beginning ``[`` and the ending ``]``.
 - Section: the lines starting from a Section Header until the beginning of
   the next Section Header or the end of the file.
+ 
+> The EditorConfig fileformat formerly allowed the use of `;` and `#` to parse the rest of a line as comment. This led to confusion how to handle values containing those characters when implementing a parser. We removed the support of inline comments entirely to make the standard easy to implement and understand. Older EditorConfig parsers may still allow inline comments.
```

Comment:
I agree we shouldn't need any poll on this. Could you perhaps update the language a bit:

> We removed the support of inline comments entirely to make the standard easy to implement and understand.

We didn't actually remove anything, we just clarified. How about we strike this sentence out?